### PR TITLE
Make Haddop and GCS API metrics to track duration as well

### PR DESCRIPTION
**Pull request by arunkumarchacko**

### CODE REVIEW

File: gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsGlobalStorageStatistics.java [hunk 1]
```diff
@@ -228,10 +230,8 @@ protected void addTotalTimeStatistic(String statistic) {
     String parentMeanKey = getMeanKey(parentCounterKey);
 
     assert (means.containsKey(parentMeanKey) && opsCount.containsKey(parentCounterKey));
```
Comment (singhravidutt) [inline gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsGlobalStorageStatistics.java:232]: we don't need counterKey assert, correct?

File: gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsGlobalStorageStatistics.java [hunk 2]
```diff
@@ -209,7 +210,8 @@ public void updateStats(GoogleCloudStorageStatistics statistic, long duration, O
   private void updateStats(
       String symbol, StatisticTypeEnum statType, long durationMs, Object context) {
     checkArgument(
-        statType == TYPE_DURATION, String.format("Unexpected instrumentation type %s", statType));
+        statType == TYPE_DURATION || statType == TYPE_DURATION_TOTAL,
```
Comment (singhravidutt) [inline gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsGlobalStorageStatistics.java:213]: add unit test case for it?

File: gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsGlobalStorageStatistics.java (hunk 1 above)
Comment (arunkumarchacko) [reply gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsGlobalStorageStatistics.java:232]: Already existing code. I guess more check wont hurt.

File: gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsGlobalStorageStatistics.java (hunk 2 above)
Comment (arunkumarchacko) [reply gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsGlobalStorageStatistics.java:213]: Done

**Review by singhravidutt — APPROVED** [review]

**Pull request merged**